infra/helper.py

```diff
@@ -419,6 +450,113 @@ def check_build(args):
   return exit_code
 
 
+def _get_fuzz_targets(project_name):
+  """Return names of fuzz targest build in the project's /out directory."""
+  command = [
+      'find',
+      _get_output_dir(project_name),
+      '-maxdepth',
+      '1',
+      '-type',
+      'f',
+      '-executable'
+  ]
+  output = subprocess.check_output(command)
+  return [os.path.basename(path) for path in output.split()]
+
+
+def _get_latest_corpus(project_name, fuzz_target, corpus_dir):
```

Comment:
nit: s/corpus_dir/base_corpus_dir/ to make it clearer.

Reply:
Good idea, done!